client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "namex",
-  "version": "2.1.11",
+  "version": "2.1.12",
```

Comment:
Please also run `npm i` and then commit the package lock file.

Reply:
it seems like the `package-lock.json` file is in the `.gitignore` file.  I'm assuming this was done on purpose for some reason.

Reply:
I think that is historical and incorrect. We want the package lock file to be checked in so that the build server uses the exact same packages that you are using locally (as recorded in the lock file).

Reply:
will add this back